v2/permissions/checker.go

```diff
@@ -28,13 +29,20 @@ func NewCheckerForStore(cache Cache) *Checker {
 	}
 }
 
+// request backoff schedule: 10/30/60 seconds
+var backoffSchedule = []time.Duration{
+	10 * time.Second,
+	30 * time.Second,
+	60 * time.Second,
+}
+
```

Comment:
Do we need to add some test cases to prove it will retry if the first response is a failure? Testing the timeout might be nasty but I think if we set up a test case where:
- The 1st check fails
- The 2nd check succeeds.

We can ignore how long it took but we can assert that the get bundle request was made 2 times - Proving that if it fails it will retry.

We should also include a test to cover the scenario where it fails all 3 attempts.

Reply:
Yes, refactored existing tests but happy to add another.